onnxruntime/core/language_interop_ops/pyop/pyop.cc

```diff
@@ -1,139 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
 #include "pyop.h"
 #ifdef _WIN32
 #define LIB_PYOP "onnxruntime_pywrapper.dll"
-#define LOAD_PYOP_LIB(n,v,m) ORT_ENFORCE((v=LoadLibraryA(n))!=nullptr,m)
+#define LOAD_PYOP_LIB(n, v, m) ORT_ENFORCE((v = LoadLibraryA(n)) != nullptr, m)
 #else
 #ifdef __APPLE__
 #define LIB_PYOP "./libonnxruntime_pywrapper.dylib"
 #else
 #define LIB_PYOP "./libonnxruntime_pywrapper.so"
 #endif
-#define LOAD_PYOP_LIB(n,v,m) ORT_ENFORCE((v=dlopen(n,RTLD_NOW|RTLD_GLOBAL))!=nullptr,m)
+#define LOAD_PYOP_LIB(n, v, m) ORT_ENFORCE((v = dlopen(n, RTLD_NOW | RTLD_GLOBAL)) != nullptr, m)
 #include "dlfcn.h"
 #endif
+#include "core/framework/tensorprotoutils.h"
 #include "core/platform/env.h"
-#define LOAD_PYOP_SYM(n,v,m) ORT_ENFORCE(Env::Default().GetSymbolFromLibrary(handle_,n,reinterpret_cast<void**>(&v))==Status::OK(),m)
+// #define LOAD_PYOP_SYM(n, v, m) ORT_ENFORCE(Env::Default().GetSymbolFromLibrary(handle_, n, reinterpret_cast<void**>(&v)) == Status::OK(), m)
 
 namespace onnxruntime {
 
 const PyOpLibProxy& PyOpLibProxy::GetInstance() {
-    static PyOpLibProxy proxy;
-    return proxy;
+  static PyOpLibProxy proxy;
```

Comment:
This is not related to your change, but I have to say, it's too dangerous and it could result a deadlock on Windows.